lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/domain/item/ReportUserIdentityItem.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.di.ipv.core.reportuseridentity.domain.item;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.persistence.item.PersistenceItem;
+
+import java.util.List;
+
+@DynamoDbBean
+@ExcludeFromGeneratedCoverageReport
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class ReportUserIdentityItem implements PersistenceItem {
+    private String userId;
+    private String identity;
+
+    @JsonProperty("constitute")
+    private List<String> constituteCriDocumentType;
```

Comment:
Should this be something like `constituentVcs` rather than `constitute`?

Reply:
done